stable/mercure/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- dunglas
```

Comment:
add the reviewers as well

Reply:
You mean @vincentchalamon and @clementtalleu? Or people from the Helm team?

Reply:
can be you as well since you are the approver